server/jetstream_api.go

```diff
@@ -4779,7 +4779,7 @@ func (s *Server) jsConsumerInfoRequest(sub *subscription, c *client, _ *Account,
 			// We have a consumer assignment.
 			if isMember {
 				js.mu.RLock()
-				if rg.node != nil {
+				if rg != nil && rg.node != nil {
```

Comment:
maybe there are other side effects from this? should we narrow down to push subscribers only?

Reply:
There are no side-effects related to this. If `rg` was nil we'd have a panic currently, so it's just guarding against that.